nncf/torch/dynamic_graph/layer_attributes_handlers.py

```diff
@@ -44,12 +44,14 @@
 
 OP_NAMES_REQUIRING_MODULE_ATTRS = [
     v.op_func_name for v in NNCF_MODULES_DICT] + list(PTGroupNormMetatype.get_all_aliases())
-OP_NAMES_REQUIRING_ATTRS_FROM_ARGS_KWARGS = list(
-    PTTransposeMetatype.get_all_aliases() + PTGatherMetatype.get_all_aliases()
-)
+
 TRANSPOSE_OP_NAMES = ['transpose', 'transpose_']
 PERMUTE_OP_NAMES = ['permute']
 GETITEM_OP_NAMES = ['__getitem__']
+GETITEM_PAD_NAMES = PTPadMetatype.get_all_aliases()
```

Comment:
Decided to change the definition of this list, because later only the function names from `TRANSPOSE_OP_NAMES`, `PERMUTE_OP_NAMES`, `GETITEM_OP_NAMES` will pass the check inside `get_layer_attributes_from_module` and all function aliases from corresponding metatypes are not needed.